tests/conftest.py

```diff
@@ -36,55 +37,28 @@
 VIRTUALENV_VERSION = Version(metadata.version("virtualenv"))
 
 
-@pytest.fixture(scope="session")
-def pep518_wheelhouse(tmp_path_factory: pytest.TempPathFactory) -> Path:
-    wheelhouse = tmp_path_factory.mktemp("wheelhouse")
+def pytest_configure(config) -> None:
```

Comment:
Can you add an opt-out flag for this for building the rpm?

I checked again on how the building is done in fedora [^1] and still no dice on making these work there.

One hing we could do is make the venv construct with `--system-site-packages` and skip all tests with build-isolation, but we would need a different interface method for  installing dependencies that can be made no-op in that case.

[^1]: https://src.fedoraproject.org/rpms/pyproject-rpm-macros/blob/rawhide/f/pyproject_wheel.py#_43